src/elfpy/markets.py

```diff
@@ -48,71 +55,32 @@ def __init__(
         self.cum_base_asset_slippage = 0
         self.cum_token_asset_fees = 0
         self.cum_base_asset_fees = 0
-        self.total_supply = self.base_asset + self.token_asset
+        self.total_supply = self.share_reserves + self.bond_reserves
         self.verbose = verbose
 
-    def apy(self, days_remaining):
-        """Returns current APY given the market conditions and pricing model"""
-        price = self.pricing_model.calc_spot_price_from_reserves(
-            self.base_asset,
-            self.token_asset,
-            self.total_supply,
-            self.time_remaining,
-            self.init_share_price,
-            self.share_price,
-        )
-        normalized_days_remaining = self.pricing_model.norm_days(days_remaining)
-        return self.pricing_model.calc_apy_from_spot_price(price, normalized_days_remaining)
-
-    def spot_price(self):
-        """Returns the current spot price given the market conditions and pricing model"""
-        return self.pricing_model.calc_spot_price_from_reserves(
-            self.base_asset,
-            self.token_asset,
-            self.total_supply,
-            self.time_remaining,
-            self.init_share_price,
-            self.share_price,
-        )
-
-    def tick(self, step_size):
-        """
-        Decrements the time variable by the provided step_size.
-
-        Arguments:
-        step_size [float] must be less than self.time_remaining
-
-        It is assumed that self.time_remaining starts at 1 and decreases to 0.
-        This function cannot reduce self.time_remaining below 0.
-        """
-
-        self.time_remaining -= step_size
-        if self.time_remaining < 0:
-            assert False, (
-                f"ERROR: the time variable market.time_remaining={self.time_remaining} should never be negative."
-                + f"\npricing_model={self.pricing_model}"
-            )
+    def base_reserves(self):
+        """get base reserves"""
+        return self.share_reserves * self.share_price
 
     def get_target_reserves(self, token_in, trade_direction):
```

Comment:
I think we may ultimately get rid of this function. Is the intention behind keeping it with "fyt" to avoid breaking more things? Regardless, I'm not concerned about it for the merge to dev.

Reply:
I believe Mihai added this for something that no longer exists. I'll remove it.